focus/commands/src/cli/main.rs

```diff
@@ -99,6 +99,10 @@ enum Subcommand {
         /// search results.
         #[clap(short = 'a', long = "all", requires("interactive"))]
         search_all_targets: bool,
+
+        /// Add the immediate targets and projects of projects to the selection, not the projects themselves.
+        #[clap(long = "unroll")]
```

Comment:
I think this can just be `clap(long)`, but it follows surrounding style, so whatever.